packages/stateful/proposal-module-adapter/adapters/CwdProposalMultiple/common/selectors.ts

```diff
@@ -0,0 +1,100 @@
+import { RecoilValueReadOnly, selectorFamily, waitForAll } from 'recoil'
+
+import { blockHeightTimestampSafeSelector } from '@dao-dao/state'
+import {
+  CheckedDepositInfo,
+  ContractVersion,
+  WithChainId,
+} from '@dao-dao/types'
+import { Status } from '@dao-dao/types/contracts/CwdProposalSingle.common'
```

Comment:
is the status type the same for both modules? if so, we may want to move this to a more general types file, even just `types/contracts/common.ts` and renaming it to `ProposalStatus` could work